frontend/public/index.html

```diff
@@ -7,6 +7,8 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&display=swap" rel="stylesheet">
```

Comment:
Is there any reason to link the font instead of downloading it/hosting it? I'm looking at https://css-tricks.com/dont-just-copy-the-font-face-out-of-google-fonts-urls/ and it seems better to download the font than hotlink it?

Reply:
mostly just to make things easier. i dont really see a disadvantage other than the request overhead, and other dti products seem to just hotlink it as well

Reply:
That's fair. Approving then